dsc/workflows/opencourseware.py

```diff
@@ -333,11 +333,13 @@ def item_metadata_iter(self) -> Iterator[dict[str, Any]]:
         ):
             yield {
                 "item_identifier": self.parse_item_identifier(file),
-                **self._extract_metadata_from_zip_file(file),
+                **self.metadata_transformer.transform(
+                    source_metadata=self._read_metadata_from_zip_file(file)
+                ),
```

Comment:
I had to sit with it, but I think this is a nice pattern of a) composing the _class_ not an _instance_ to `self`, then for each iteration here b) passing the source metadata to the class method `transform()`.  

Having them all be class methods probably makes them a _little_ trickier to work with (e.g. that do I want source metadata or not bit), but it makes it much simpler at this level.

Reply:
Yeah! I've been thinking of the `OpenCourseWareTransformer` as simply a collection of methods at its current state, since instantiation is not required. 🤔